aries-site/src/examples/templates/dashboards/components/ActivityFeed.js

```diff
@@ -18,9 +18,9 @@ export const ActivityFeed = ({ children, title, footer: footerContent }) => {
       <CardHeader
         pad={{ horizontal: header.pad, top: header.pad, bottom: 'small' }}
       >
-        <Text weight="bold" size="xlarge" color="text-strong">
+        <Heading margin="none" size="small" level={2}>
```

Comment:
```suggestion
        <Heading margin="none" level={2}>
```
With the next theme release, we are going to lean away from using any `size` prop on Heading. Removing here so we don't have any debt to go back and remove.